assets/js/base/hooks/shipping/use-shipping-address.js

```diff
@@ -39,8 +40,16 @@ export const useShippingAddress = () => {
 			);
 		}
 	}, [ debouncedShippingAddress ] );
+
+	const decodedShippingAddress = {};
+	Object.keys( shippingAddress ).forEach( ( key ) => {
+		decodedShippingAddress[ key ] = decodeEntities(
+			shippingAddress[ key ]
+		);
+	} );
+
```

Comment:
we should start investing in a `fromEntities`  pollyfill later

Reply:
Agree, I missed being able to use it here. :smile: